backend-service/src/main/java/com/smartparkinglot/backendservice/controller/TicketController.java

```diff
@@ -30,7 +32,16 @@
     public ResponseEntity<Ticket> addOrSave(@RequestBody Ticket ticket){
         return new ResponseEntity<Ticket>(ticketService.addOrSave(ticket),HttpStatus.CREATED);
     }
+
+    @PostMapping("rate-parking-lot")
+    @Operation(summary = "send ticket with rate on it")
+    public ResponseEntity<Ticket> rateTicket(@RequestBody Ticket ticket){
+        return new ResponseEntity<Ticket>(ticketService.rateTicket(ticket),HttpStatus.CREATED);
+    }
+
+
     @DeleteMapping
+    @Operation(summary = "DONT USE it for deletion")
```

Comment:
[This Comment](https://github.com/bitirme-projesi-beu/backend-service/pull/14/files#r433806525) includes this one too.